test.js

```diff
@@ -25,23 +28,46 @@ test('falsey cases', t => {
 		'NO',
 		'No',
 		'false',
+		'FALSE',
+		'False',
 		false,
+		'0',
 		0
 	].forEach(el => {
 		t.false(yn(el));
 		t.false(yn(el, {lenient: true}));
 	});
 });
 
-test('edge cases which return null', t => {
-	t.is(yn(NaN), null);
-	t.is(yn(''), null);
-	t.is(yn('yn'), null);
-	t.is(yn('unicorn'), null);
-	t.is(yn(NaN, {lenient: true}), null);
-	t.is(yn('', {lenient: true}), null);
-	t.is(yn('yn', {lenient: true}), null);
-	t.is(yn('unicorn', {lenient: true}), null);
+test('null cases', t => {
+	[
```

Comment:
Why? The test works correctly afaict; when I change the code to
```javascript
	].forEach(el => {
		console.log(`Testing yn(${el}) === null`);
		t.is(null, yn(el));
		t.is(null, yn(el, {lenient: true}));
	});
```
all 19 test cases show up in the console.
(I don't mean to sound obnoxious, I'm just new to javascript and want to understand)